ui/app/components/ui/list-item/index.scss

```diff
@@ -0,0 +1,84 @@
+.list-item {
+  width: 100%;
+  min-width: 374px;
+  min-height: 86px;
+  margin: 0 20px;
+  background: #FFFFFF;
+  padding: 24px 16px;
+  @extend %font;
+  border-top: 1px solid $mercury;
+  border-bottom: 1px solid $mercury;
+  color: $Grey-500;
+
+  display: flex;
+  justify-content: flex-start;
+  align-items: stretch;
+
+  &__icon {
+    margin: 8px 14px 0 0;
+  }
+
+  &__col {
+    align-self: flex-start;
+  }
+
+  &__more {
+    &--button {
```

Comment:
These `--button` styles don't belong here, since the component itself doesn't use them.

Maybe you could add them inline in the storybook? As needed anyway - some of these can probably be dropped.